docs/infrastructure/logging.md

```diff
@@ -257,6 +264,116 @@ determines how often logger should send its collected events to sentry.io
 service. If `debug` value is `true`, logging will print to stdout additional
 trace information of the logging process itself.
 
+## Logging server
+
+The centralised logging service is implemented as a logging server started by
```

Comment:
```suggestion
The centralized logging service is implemented as a logging server started by
```
We should be consistent in spelling of the "central..." words. US spelling is preferred as far as I know.